main.go

```diff
@@ -49,6 +57,28 @@ func startRun(config *common.CatalogConfig, rh request.Handler) {
 	rh.StartHandlingRequests(config, &towerapiworker.DefaultAPIWorker{})
 }
 
+func getConfigFile() (string, error) {
+	if fileExists("./rhc/workers/catalog.toml") {
+		return "./rhc/workers/catalog.toml", nil
+	}
+	usr, err := user.Current()
+	if err == nil && fileExists(fmt.Sprintf("%s/.config/rhc/workers/catalog.toml", usr.HomeDir)) {
```

Comment:
@bzwei Should we just a variable to do store the filename once from the Sprintf and then use it in 2 places.